ghcide.cabal

```diff
@@ -187,6 +190,7 @@ executable ghcide
         Paths_ghcide
 
     default-extensions:
+        DeriveGeneric
```

Comment:
This was added here because HLint complains when I added it to `Main.hs`:

```
./exe/Main.hs:7:1: Warning: Avoid restricted extensions
Found:
  {-# LANGUAGE DeriveGeneric #-}
Note: may break the code
```

Reply:
Indeed, that's the right thing to do. At Digital Asset we made a list of extensions we felt were "boring", and thus should never occur in a .hs file, only in default-extensions or other places.